qvm/sys-audio.sls

```diff
@@ -24,10 +24,33 @@ prefs:
   - pci_strictreset: false
   - pcidevs:   {{ salt['grains.get']('pci_audio_devs', [])|yaml }}
   - template:  {{ salt['pillar.get']('qvm:sys-audio:template', 'fedora-30-xfce') }}
+service:
+  - enable:
+    - audiovm-audio-agent
 {%- endload %}
 
 {{ load(defaults) }}
 
+# Setup Qubes RPC policy for sys-audio
+# If sys-audio is sys-gui it's not needed
+# as sys-gui is a admin-global-rwx
+sys-audio-rpc:
+  file.managed:
+    - name: /etc/qubes/policy.d/30-sys-audio.policy
+    - contents: |
+        admin.Events          *   sys-audio     sys-audio               allow
```

Comment:
All `admin.*` policy lines needs `target=dom0`. Technically those with `@adminvm` target will work also without it, but I think it's better to add there too for consistency.

Reply:
Done.